docs/server/commands.rst

```diff
@@ -324,6 +324,9 @@ Available options:
 
 .. django-admin-option:: --from-revision
 
+  .. versionchanged:: 2.8.0 You cannot specify this option and
```

Comment:
TBH from a users perspective it is way more important what it does, then the fact that it is mutually exclusive.  That is lost with the noise that @julen pointed out.